buildpacks/nodejs-engine/Cargo.toml

```diff
@@ -26,6 +26,8 @@ toml = "0.8"
 insta = "1"
 libcnb-test = "=0.28.1"
 regex = "1"
+reqwest = "0.12"
+reqwest-middleware = "0.4"
 serde_json = "1"
 test_support.workspace = true
-ureq = "3"
+tokio = "1"
```

Comment:
Suggestion: There is a lot of repetition in the version of all these cargo.toml files. I suggest we move them to the workspace top level `Cargo.toml` and then these would become `tokio.workspace = true }`. It will make future version bumps easier and their diffs smaller, and prevent a scenario where common and a buildpack accidentally get out of sync.

This isn't a blocker, we could move that to the backlog to do an audit update the rest at once.

Reply:
We did have that at one point until @edmorley found that using workspace dependencies wouldn't work right with automated Dependabot updates. I believe this is still an issue as https://github.com/dependabot/dependabot-core/issues/7896 is still open.